src/Features/Core/Portable/ConvertAnonymousType/AbstractConvertAnonymousTypeToTupleCodeRefactoringProvider.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
this was a rename/refactoring of https://github.com/dotnet/roslyn/pull/50536/files#diff-4d2eaffed8275d428f2012b7cb6e29a42b5e5a04409674c480cec2d99054fbdcL1, but github doesn't seem to have picked that up.